src/App.tsx

```diff
@@ -1,14 +1,26 @@
-
-import './App.css'
+import { Routes, Route } from 'react-router-dom';
+import Layout from './components/Layout/Layout';
+import Home from './pages/Home/Home';
```

Comment:
please create `index.js` file in `/pages` folder and then import pages from there. the same applies for every other folder that contains more export - for instance, components folder.